src/cmd/manage_nsfs.js

```diff
@@ -690,6 +674,56 @@ async function get_config_data(config_file_path, show_secrets = false) {
     return config_data;
 }
 
+/**
+ * get_options_from_file will read a JSON file that include key-value of the options 
+ * (instead of flags) and return its content
+ * @param {string} type
+ * @param {string} action
+ * @param {string} file_path
+ */
+async function get_options_from_file(type, action, file_path) {
+    const fs_context = native_fs_utils.get_process_fs_context();
+    try {
+        const raw_data = (await nb_native().fs.readFile(fs_context, file_path)).data;
```

Comment:
can you please use native_fs_utils.read_config_file() and change the function name?

Reply:
I will change.